ginkgo_me/BBQ.go

```diff
@@ -1,10 +1,10 @@
 package ginkgo_me
 
 import (
-	beeffarm "test_me/pkgs/beef-farm"
-	chickenfarm "test_me/pkgs/chicken-farm"
-	coalfarm "test_me/pkgs/coal-farm"
-	mangalstore "test_me/pkgs/mangal-store"
+	beeffarm "github.com/artiomgiza/test_me/pkgs/beef-farm"
+	chickenfarm "github.com/artiomgiza/test_me/pkgs/chicken-farm"
+	coalfarm "github.com/artiomgiza/test_me/pkgs/coal-farm"
+	mangalstore "github.com/artiomgiza/test_me/pkgs/mangal-store"
```

Comment:
do you know why this is like this?
looks like your is righter, interesting why mine is relative...

Reply:
your way didn't work for me :(